content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -403,6 +410,27 @@ In the past, the annotation `volume.beta.kubernetes.io/mount-options` was used i
 of the `mountOptions` attribute. This annotation is still working, however
 it will become fully deprecated in a future Kubernetes release.
 
+### Node Affinity
+
+A PV can specify [node affinity](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volumenodeaffinity-v1-core) to define constraints that limit what nodes this volume can be accessed from. Only the nodes that satifsy the constraints can access the volume.
+
+Node affinity is conceptually similar to NodeSelector.
+
+NodeSelector is specified as field `required` of field `nodeAffinity` in the PersistentVolumeSpec.
+
+A node selector can have one or more [node selector terms](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#nodeselectorterm-v1-core).
+
+If you specify multiple `nodeSelectorTerms`, then a node can access the volume **if one of** the `nodeSelectorTerms` is satisfied. A node selector term can consist of two fields
+
+* `matchExpressions` - a list of requirements by node's labels
+* `matchFields` - a list of requirements by node's fields
+
+All of the requirements, from both `matchLabels` and `matchExpressions` are ANDed together – they must all be satisfied in order to match.
+
+In the past, the annotation `volume.alpha.kubernetes.io/node-affinity` was used instead
+of the `nodeAffinity` attribute. This annotation is still working, however
```

Comment:
This is incorrect.  We removed alpha annotation support in 1.11 I think.

Reply:
Updated to beta